bridge/src/main/java/io/stargate/bridge/service/QueryHandler.java

```diff
@@ -154,14 +154,25 @@ private ByteBuffer getComparableBytesFromRow(
   }
 
   private ByteBuffer getPagingStateFromRow(
-      ByteBuffer resultSetPagingState, Row row, boolean lastInPage) {
+      ByteBuffer resultSetPagingState,
+      Row row,
+      QueryOuterClass.ResumeMode resumeMode,
+      boolean lastInPage) {
     if (lastInPage && resultSetPagingState == null) {
       return EXHAUSTED_PAGE_STATE;
     }
 
+    if (resumeMode == null) {
+      return null;
```

Comment:
also questioning if this `null`, will throw `NullPointerException` once you try to set it to the row builder? Is this taken care of?

Reply:
Yea this is handled in ValuesHelper (line 213)